app/javascript/components/App.js

```diff
@@ -20,26 +20,107 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      parties: [],
-      items: [],
+      party_id: '',
+      parties: '',
+      items: '',
```

Comment:
best practice would be initialize these values with the a value of the same data type as the actual data or use `undefined`. 
```suggestion
      party_id: undefined,
      parties: [],
      items: [],
```